### DISCUSSION TITLE
video doesn't load

### DISCUSSION THREAD

**Discussion by anthony2025** [community/community#10549] [Other Feature Feedback, Questions, & Ideas] [answered]
Going to the video url https://cs.github.com/about/video on Firefox shows an empty gray window with:

`Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”).`
`The resource from “https://cs.github.com/static/js/main.e3595bc2.js” was blocked due to MIME type (“text/plain”) mismatch (X-Content-Type-Options: nosniff). video`

**Comment by colinwm** [comment, marked as answer]
Great catch! Thanks for the tip, this is now fixed
